test/cbmc/include/core_sntp_stubs.h

```diff
@@ -92,6 +92,30 @@ SntpStatus_t GenerateClientAuthStub( SntpAuthContext_t * pContext,
                                      size_t bufferSize,
                                      size_t * pAuthCodeSize );
 
+/**
+ * @brief Application defined function to authenticate server by validating
+ * the authentication code present in its SNTP response to a time request, if
+ * a security mechanism is supported by the server.
+ *
+ * @param[in,out] pContext The application defined NetworkContext_t which
+ * is opaque to the coreSNTP library.
+ * @param[in] pTimeServer The time server that has to be authenticated from its
+ * SNTP response.
+ * @param[in] pResponseData The SNTP response from the server that contains the
+ * authentication code after the first #SNTP_PACKET_BASE_SIZE bytes.
+ * @param[in] responseSize The total size of the response from the server.
+ *
+ * @return The function SHOULD return one of the following integer codes:
+ * - #SntpSuccess when the server is successfully authenticated.
```

Comment:
These status values are not relevant as this function implementation always returns success, right?

Reply:
Yeah, it will always return Success.

Reply:
Can you clearly mention that this function will always return success? 
Similarly, can you update the documentation of the stub for `SntpGenerateAuthCode_t` function as well?

Reply:
SntpGenerateAuthCode_t documentation in the stub seems to be correct to me as it return both the values that are being mentioned over there. Is there anything specific you want to me to update?